src/main/java/com/solvd/block3/ui/UiUtils.java

```diff
@@ -92,34 +91,34 @@ public static boolean validateCity(String cityId) {
     }
 
 
-    public static String getMode()
-    {
-        LOGGER.info("Enter 's' to get the shortest path or 'c' for the cheapest");
+    public static String getMode() {
+        LOGGER.info("Enter 's' to get the shortest route or 'c' for the cheapest");
         String mode = SCANNER.nextLine().trim();
-        while (!mode.equalsIgnoreCase("s") && !mode.equalsIgnoreCase("c"))
-        {
-            LOGGER.info("Invalid input.");
-            LOGGER.info("Enter 's' to get the shortest path or 'c' for the cheapest");  
-            mode = SCANNER.nextLine().trim();
-        }
 
-        return mode;
+        // Use regex to match variations of input
```

Comment:
Added a pattern to match potential variations of user input for the cheapest and shortest flight options.